### PR TITLE
Fix rpath issues on mac for libmi.dylib

### PR DESCRIPTION
**Pull request by paulcallen**
@Microsoft/omi-devs

### CODE REVIEW

File: Unix/installbuilder/GNUmakefile [hunk 1]
```diff
@@ -133,25 +133,26 @@ ifneq ($(PF),Darwin)
 
 	mkdir -p $(PACKAGE_DIR)
 	cp $(OUTPUTDIR)/release/omi-* $(PACKAGE_DIR)
+endif # ifneq ($(PF),Darwin)
 
-else # ifneq ($(PF),Darwin)
-
+ifeq ($(PF),Darwin)
```
Comment (jeffaco) [inline Unix/installbuilder/GNUmakefile]: I'm not sure what the difference is here, or was this just a cosmetic change for you?

File: Unix/installbuilder/GNUmakefile [hunk 2]
```diff
@@ -133,25 +133,26 @@ ifneq ($(PF),Darwin)
...
 	mkdir -p $(PACKAGE_DIR)
 	cp $(OUTPUTDIR)/bin/omicli $(PACKAGE_DIR)
 
-endif # ifneq ($(PF),Darwin)
-
-	# We also need a raw copy of libmi shared library. If libmi_origin exists
-	# use that and rename it libmi.
+	# copy the libmi_origin.dylib to libmi.dylib 
+	cp $(OUTPUTDIR)/lib/libmi_origin.dylib $(PACKAGE_DIR)/libmi.dylib
+else #($(PF),Darwin)
```
Comment (jeffaco) [inline Unix/installbuilder/GNUmakefile]: Prefer a full comment after the 'else'. In this case, `ifeq`. `ifeq ($(PF),Darwin)` is a lot different than `ifneq ($(PF),Darwin)`.

File: Unix/installbuilder/GNUmakefile (hunk 1 above)
Comment (paulcallen) [inline Unix/installbuilder/GNUmakefile]: yes, cosmetic

File: Unix/installbuilder/GNUmakefile (hunk 2 above)
Comment (paulcallen) [inline Unix/installbuilder/GNUmakefile]: these were very close to each other and so that is why I chose not to, but I am happy to add it

**Pull request merged**